prow/config.yaml

```diff
@@ -3695,14 +3697,15 @@ periodics:
     containers:
     - image: gcr.io/k8s-testimages/bootstrap:v20180730-c3257f6c0
       args:
-      - "--repo=k8s.io/kubernetes=release-1.10"
+      - "--repo=k8s.io/kubernetes=release-1.11"
```

Comment:
Do we need a `"--timeout=75"` here like there is elsewhere?

Reply:
done